core/src/main/scala/com/salesforce/op/OpWorkflowModelWriter.scala

```diff
@@ -138,7 +139,7 @@ private[op] object OpWorkflowModelReadWriteShared {
     case object AllFeatures extends FieldNames("allFeatures")
     case object Parameters extends FieldNames("parameters")
     case object TrainParameters extends FieldNames("trainParameters")
-    case object RawFeatureDistributions extends FieldNames("rawFeatureDistributions")
```

Comment:
Great question!

I think by design, `FieldNames` refers to the set of field names being saved by `OpWorkflowModelWriter` in all models going forward.
- First unit test in `OpWorkflowModelReaderWriterTest` checks whether newly saved models have a set of field names that are identical to those in `FieldNames`. If we bring in `RawFeatureDistributions` into `FieldNames`, it will fail the unit test, unless we also choose to continue writing `RawFeatureDistributions` (would be redundant)

Also, I think backwards compatibility shouldn't be in the Writer, but should be contained (as much as possible) in the Reader. Current design does this. Backwards compatibility is maintained through the `resolveRawFeatureDistributions()` in `OpWorkflowModelReader`.

Reply:
Ok, that makes sense